### PR TITLE
(DOCSP-26322) Added an IP access list restriction

### PR DESCRIPTION
**Pull request by JuliaMongo**
[DOCSP-26322](https://jira.mongodb.org/browse/DOCSP-26322)

@jiayong0 and @tsck, could you please review?

[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-26322/connect-atlas-cli/#connect-with-minimum-required-settings) (choose the "non-programmatic" tab)

[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=63f4ebd105dd3cc814caf887)

@sarahsimpers, this must be merged and published only after this feature branch for Atlas gets merged and published ([ip-access-list-full](https://github.com/10gen/cloud-docs/tree/ip-access-list-full)).

### CODE REVIEW

File: source/connect-atlas-cli.txt [hunk 1]
```diff
@@ -96,6 +96,16 @@ Complete the Prerequisites
 
       - :ref:`Install the {+atlas-cli+} <install-atlas-cli>`.
       - Add your host's IP address to the :ref:`IP access list <access-list>`.
+  
+        If you log into the {+atlas-cli+} with your Atlas user credentials,
```
Comment (sarahsimpers) [inline source/connect-atlas-cli.txt:100]: Nit: they don't log in to the CLI, they log in to Atlas using the CLI. Also, no comma needed here
```suggestion
        If you authenticate with your |service| user credentials
```

File: source/connect-atlas-cli.txt [hunk 2]
```diff
@@ -96,6 +96,16 @@ Complete the Prerequisites
...
+        and your organization's owners enable
+        :atlas:`IP access list for the Atlas UI for an organization
+        </tutorial/manage-organizations/#require-ip-access-list-for-the-atlas-ui>`,
+        then to run any commands in this organization, your IP address must
+        be included in the IP access list or CIDR ranges defined in that list.
```
Comment (sarahsimpers) [inline source/connect-atlas-cli.txt:105]: Suggestion: simplifies the text here ("must be added" could be either as a single IP address or CIDR range)
```suggestion
        your IP address must be added to the IP access list to run 
        commands in this organization.
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
A few small changes, I think you put this in the right place.

For the submodule: I don't actually think there's a problem since [the compare](https://github.com/mongodb/mongodb-atlas-cli/compare/a3d64fc13b7ec5742f5b325f32eaf53a4cc0447a...39507b21e19734762cdedb19e9e7c7b7b5ee4c5a) shows that no files are changing in this update, but I'm not 100% sure, so let me ask around before approving.

**Review by sarahsimpers — APPROVED** [review]
Per conversation on Slack, updates to the SHA are expected so this is good to go! Thanks!

File: source/connect-atlas-cli.txt [hunk 3]
```diff
@@ -96,6 +96,14 @@ Complete the Prerequisites
 
       - :ref:`Install the {+atlas-cli+} <install-atlas-cli>`.
       - Add your host's IP address to the :ref:`IP access list <access-list>`.
+  
+        If you authenticate with your |service| user credentials
+        and your organization's owners enable :atlas:`IP access list for the Atlas UI for an organization
+        </tutorial/manage-organizations/#require-ip-access-list-for-the-atlas-ui>`,
+        your IP address must be added to the IP access list to run commands
+        in this organization. To learn more, see :atlas:`Require IP Access List for the Atlas UI
+        </tutorial/manage-organizations/#require-ip-access-list-for-the-atlas-ui>`.
```
Comment (jiayong0) [inline source/connect-atlas-cli.txt:105]: Not sure if this is a config issue, but when clicked this link it actually took me to [here](https://www.mongodb.com/docs/atlas/tutorial/manage-organizations/?_ga=2.25262141.1031606856.1676317638-1380600391.1667244526#require-ip-access-list-for-the-atlas-ui), but the page didn't have the instructions on how to set up ip access list. I remembered UI IP access list related doc only lives in staging and not publicly accessible?

**Review by tsck — APPROVED** [review]
Other than Jiayong's comment, LGTM!

File: source/connect-atlas-cli.txt (hunk 3 above)
Comment (JuliaMongo) [reply source/connect-atlas-cli.txt:105]: Yes. This link won't work until both PRs are published. I will not merge or publish either IP access list PRs until the feature goes GA. So this link will work once things are published.

Comment (JuliaMongo) [reply source/connect-atlas-cli.txt:105]: Please approve. I will keep the PR in the Ready For Merge state until the feature goes live.

**Review by jiayong0 — APPROVED** [review]

**Pull request merged**